chapters/ch31.md

```diff
@@ -1,32 +1,32 @@
 > # 31. Interpreting learning curves: Other cases
 
---> _replace THIS LINE by your translation for the above line_
+# 31. Giải nghĩa các đồ thị quá trình học: Những trường hợp khác
 
 
 > Consider this learning curve:
 
---> _replace THIS LINE by your translation for the above line_
+Hãy xem xét đồ thị quá trình học này:
 
 
 ![img](../imgs/C31_01.png)
 
 > Does this plot indicate high bias, high variance, or both?
 
---> _replace THIS LINE by your translation for the above line_
+Đồ thị này thể  hiện độ chệch lớn, phương sai lớn hay cả hai?
 
 
 > The blue training error curve is relatively low, and the red dev error curve is much higher than the blue training error. Thus, the bias is small, but the variance is large. Adding more training data will probably help close the gap between dev error and training error.
 
---> _replace THIS LINE by your translation for the above line_
+Đường cong lỗi huấn luyện màu xanh tương đối thấp và đường cong lỗi phát triển màu đỏ cao hơn nhiều so với lỗi huấn luyện màu xanh. Do đó, độ chệch nhỏ, nhưng phương sai lớn. Thêm dữ liệu huấn luyện (có khả năng lớn) sẽ giúp thu hẹp khoảng cách giữa lỗi phát triển và lỗi huấn luyện.
 
 
 > Now, consider this:
 
---> _replace THIS LINE by your translation for the above line_
+Bây giờ, hãy xem xét đồ thị này:
 
 
 ![img](../imgs/C31_02.png)
 
 > This time, the training error is large, as it is much higher than the desired level of performance. The dev error is also much larger than the training error. Thus, you have significant bias and significant variance. You will have to find a way to reduce both bias and variance in your algorithm.
 
---> _replace THIS LINE by your translation for the above line_
+Lần này, lỗi huấn luyện lớn, vì nó cao hơn nhiều so với mức hiệu suất mong muốn. Lỗi phát triển cũng lớn hơn nhiều so với lỗi huấn luyện. Vì vậy, bạn có độ chệch đáng kể  và phương sai cũng đáng kể. Bạn sẽ phải tìm cách giảm cả độ chệch và phương sai trong thuật toán của mình.
```

Comment:
* "Performance" được dịch là "hiệu quả" từ đầu cuốn sách. Bây giờ mình dịch chung như thế cho nhất quán. Sau nếu muốn sửa lại chỉ cần search and replace là được.

* Hai dấu cách "kể  và phương sai"